.github/workflows/pre-release.yml

```diff
@@ -14,10 +14,12 @@ jobs:
       pull-requests: write
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
+
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
```

Comment:
this change makes the file invalid. The `uses` key should be part of the same object that starts with `- name: Checkout Repo` , or remove the `name` key all together and just leave the `uses`